From 8a704321efc975a96cb1bdc49f8ff715cd11440d Mon Sep 17 00:00:00 2001
From: Shunsuke Kataoka <shunsuke.kataoka.df@renesas.com>
Date: Fri, 21 Jun 2013 14:34:12 +0900
Subject: [PATCH 0654/1083] ARM: shmobile: r8a7790: Add sysdma lower channel
 driver support

Add sysdmal_device to r8a7790_early_devices.

Signed-off-by: Shunsuke Kataoka <shunsuke.kataoka.df@renesas.com>
(cherry picked from commit d07fbeeacd31aa25e4e17167d3a7d25288969430)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 arch/arm/mach-shmobile/setup-r8a7790.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-shmobile/setup-r8a7790.c b/arch/arm/mach-shmobile/setup-r8a7790.c
index 6f0bd4f..91467ba 100644
--- a/arch/arm/mach-shmobile/setup-r8a7790.c
+++ b/arch/arm/mach-shmobile/setup-r8a7790.c
@@ -1332,6 +1332,7 @@ static struct platform_device *r8a7790_early_devices[] __initdata = {
 	&audmal_device,
 	&audmau_device,
 	&audmapp_device,
+	&sysdmal_device,
 	&sysdmau_device,
 	&alsa_soc_platform_device,
 	&scu_device,
-- 
1.8.3.2

